### PR TITLE
Ny times

### PR DESCRIPTION
**Pull request by Henryhaulka**
In this project I have:

- Cloned the  New York Times article page.

- Embedding images

- Made use of the box model.

- Made use of Flexbox.

- Made use of Grid.

### CODE REVIEW

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
# **Changes Required 🛑♻**

<hr>

Hello there, I hope you're doing fine. Great work you've done so far on this project. It's pretty much like the original page. However, there are still a few improvements to make to your project in order to get it complete 😊🙏

- [x] Please make sure to write a descriptive Pull Request (PR) summary (get a quick refresher from [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)
- [x] Please make sure to write a descriptive README file (also follow the instructions [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)
- [x] For best practices. Kindly don't leave any anchor tag (aka ```<a href="#">```) href blank, always put a value in it i.e. "#".
- [x] Your Navigation Bar should be in a fixed position, please re-style that part in your CSS file to match the original page.
- [x] In your CSS file, there is some repeated code. Please follow the DRY rule (Don't Repeat Yourself) It's a very important and common rule that every programmer follows. i.e. Instead of 
```
.section15 a {
  text-decoration: none;
}

.nav-right p a {
  color: black;
  text-decoration: none;
}

section h4 a:hover {
  text-decoration: none;
}
```
You could've done it like this
```
.section15 a,
.nav-right p a,
section h4 a:hover {
  text-decoration: none;
}

.nav-right p a {
  color: black;
}
```

- [x] I don't know why you've left the left side of the page next to the grid blank! Would you please make it like the original page using grid, float, and flex CSS styling techniques. As well as the footer of the page (I see that you've missed that part as well)
- [x] Please follow the Git Flow Guidelines and fix your linter errors in order for your PR to pass the Github checks.

<hr>

Kindly do the above changes and submit them for another review.
Good luck and happy coding 👨‍💻💻😊

**Review by Temmarie — CHANGES_REQUESTED** [review]
## Changes Required! :recycle:

### Hi @Henryhaulka     ! :wave: :wave:

#### You have done well so far :clap: However, there are changes required for you to make in order for this project to be approved. Kindly work on them and submit them for another code review.

---

### **Recommended**

**General**

- [x] You have to write a descriptive Pull Request Message with a descriptive title as well. It's essential that your repository is professional, as this is one of the things that potential colleagues will look at when recruiting team members. Please refer to [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) for more information.

**Section Specific**

- [x] Navigation bar
-  The navbar needs a bit more work done on it, you are missing the login button, the hamburger icon is too far to the left and you have a space between the top of the page and the navbar. Please compare to the actual site and make the changes.

**Original** |  **Your clone**
------------ | -------------
Screenshot (73) | Screenshot (86)

- [x] Footer
-  The footer is just about ready, just reduce the font size used in this section.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1058 @@
...
+            </a>  
+            <p>
+                <a href="#">
+                    Go to Home Page<i class="fas fa-angle-double-right"></i>
+                </a>
+            </p>
+        </div>
+        <div class="section30">
+            <div class="section31">
+                <a href="#" style="font-weight: bold;">
```
Comment (andresporras3423) [inline index.html]: Remove inline styles

File: style.css [hunk 2]
```diff
@@ -0,0 +1,718 @@
...
+  margin: auto;
+}
+
+#inside {
+  width: 15%;
+  border-top: 1px solid grey;
+  margin: auto;
+}
+
+#insider {
```
Comment (andresporras3423) [inline style.css]: Don't overuse ID selectors

File: style.css [hunk 3]
```diff
@@ -0,0 +1,718 @@
...
+.nav-left {
+  display: flex;
+  padding-top: 1rem;
+  justify-content: space-between;
+  width: 20%;
+}
+
+footer a {
+  text-decoration: none;
+}
```
Comment (andresporras3423) [inline style.css]: Group selectors on lines 36 and 46

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Henry Onu** 
Good job with the improvements for this project :muscle:  :muscle:
Your page really looks like the original :clap: :clap: :clap:  
There are just a few details to improve before you can go ahead:

- [ ] Don't use inline styles in your HTML file
- [ ] Group selectors with the same style definition. Check out this link: http://www.websiteoptimization.com/speed/tweak/groupsel/
- [ ] Don't overuse ID selectors
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1068 @@
...
+                    <i class="fas fa-calendar-day"></i>
+                    <a href="#">
+                        <span> 
+                            Home Delivery 
+                       </span>
+                    </a>
+                </div>
+                <div class="sub-section2" >
+                    <img src="https://i.pinimg.com/originals/5e/af/20/5eaf20e1a08c73f59d40ab3ba5bffaae.png" alt="logo">
+                    <a href="#" style="margin-top: 5px;">
```
Comment (SmithVyne) [inline index.html]: Try to avoid using inline styling.

**Review by SmithVyne — CHANGES_REQUESTED** [review]
### Changes required 🤕 👍 
Hello, you have almost completed this project you just need to make the following additional changes to get it approved:
- [REQUIRED] - Do not use inline styles. Check my comment for the exact line where you did.

Make these changes and submit them for a re-review you should get the project approved this time.

You can also reach out to me on slack, @SmithVyne if you have any other questions.

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Henry Onu** 
Good job with the improvement for this project :muscle:  :muscle:
Your project is almost perfect :clap: :clap: :clap:  
Just one last detail to improve before you can go ahead:

- [x] Is a general requirements for HTML/CSS projects to use efficient selectors. Remember the DRY principle (don't repeat yourself). You have more than one instance of several selectors with the same style definition. Please, group selectors wherever possible, check out this link: http://www.websiteoptimization.com/speed/tweak/groupsel/
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

**Review by SpaYco — APPROVED** [review]
# Project Approved ✔️ 

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**